wiki/Releases/4.19.0.md

```diff
@@ -48,10 +57,22 @@ title: rpm.org - Releases
 #### Package format
 * Populate user and group data in cpio payload again (ancient regression)
 
+### Configuration
```

Comment:
I'm not sure if this is the right heading to use for these changes, any better ideas?

Reply:
Right, thanks, I'll update that.